specification/servicefabricmanagedclusters/ServiceFabricManagedClusters.Management/NodeType.tsp

```diff
@@ -904,14 +904,14 @@ model VMSSExtensionProperties {
   /**
    * Json formatted public settings for the extension.
    */
-  #suppress "@azure-tools/typespec-azure-resource-manager/no-empty-model" "Day 0 property"
-  settings?: {};
+  #suppress "@azure-tools/typespec-azure-resource-manager/arm-no-record" "Day 0 property"
+  settings?: Record<unknown>;
 
   /**
    * The extension can contain either protectedSettings or protectedSettingsFromKeyVault or no protected settings at all.
    */
-  #suppress "@azure-tools/typespec-azure-resource-manager/no-empty-model" "Day 0 property"
-  protectedSettings?: {};
+  #suppress "@azure-tools/typespec-azure-resource-manager/arm-no-record" "Day 0 property"
+  protectedSettings?: Record<unknown>;
```

Comment:
A notice that I found the unmerged migration of compute (which contains VMSS) uses `unknown`.

PR https://github.com/Azure/azure-rest-api-specs/pull/32748
tsp
https://github.com/Azure/azure-rest-api-specs/blob/710a6e41bd2d2a659615833dfebdf67a22fb0732/specification/compute/Compute.Management/models.tsp#L2989-L2999

Loop @pshao25 for advise too.